src/DssSpell.sol

```diff
@@ -65,312 +72,282 @@ contract DssSpellAction is DssAction {
-
-        // Kohla (Cloaky) - 10000 DAI - 0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a
-        DssExecLib.sendPaymentFromSurplusBuffer(CLOAKY_KOHLA_2, 10_000);
-
-        // Ennoia (Cloaky) - 10000 DAI - 0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b
-        DssExecLib.sendPaymentFromSurplusBuffer(CLOAKY_ENNOIA, 10_000);
-
-        // Bonapublica - 8333 DAI - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        DssExecLib.sendPaymentFromSurplusBuffer(BONAPUBLICA, 8_333);
-
-        // Rocky - 7796 DAI - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        DssExecLib.sendPaymentFromSurplusBuffer(ROCKY, 7_796);
-
-        // ---------- Aligned Delegate MKR Compensation ----------
-        // Forum: https://forum.sky.money/t/september-2024-aligned-delegate-compensation/25489
-        // Atlas: https://sky-atlas.powerhouse.io/A.1.5.8_Budget_For_Prime_Delegate_Slots/e3e420fc-9b1f-4fdc-9983-fcebc45dd3aa%7C0db3af4ece0c
-
-        // BLUE - 13.75 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 13.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Cloaky - 29.25 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 29.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // JuliaChang - 28.75 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 28.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Byteron - 9.68 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 9.68 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // vigilant - 2.43 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
-        MKR.transfer(VIGILANT, 2.43 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Bonapublica - 2.06 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        MKR.transfer(BONAPUBLICA, 2.06 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Rocky - 1.17 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 1.17 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Forum: https://forum.sky.money/t/utilization-of-the-integration-boost-budget-a-5-2-1-2/25536/2
+        // Poll: https://vote.makerdao.com/polling/QmYHUDVA
+
+        // Transfer 3,000,000 USDS to 0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7
+        _transferUsds(INTEGRATION_BOOST_INITIATIVE, 3 * MILLION * WAD);
+
+        // ---------- ESM Minimum Threshold increase ----------
+        // Forum: https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579
+
+        // Increase ESM Minimum Threshold by 200,000 MKR, from 300,000 MKR to 500,000 MKR
+        DssExecLib.setValue(MCD_ESM, "min", 500 * THOUSAND * WAD);
+
+        // ---------- October 2024 AD Compensation ----------
```

Comment:
Please add the [missing Authority URL](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=485154570#gid=485154570&range=H101) to the Atlas to this item

Reply:
Done